unearthed-arcana/2021/20210608.xml

```diff
@@ -0,0 +1,1397 @@
+			<grant type="Archetype Feature" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_SILVERY_BARBS" />
+			<grant type="Archetype Feature" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_INKY_SHROUD" level="6" />
+			<grant type="Archetype Feature" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_INFUSION_OF_ELOQUENCE" level="10" />
+			<grant type="Archetype Feature" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_WORD_OF_POWER" level="14" />
+		</rules>
+	</element>
+
+	<element name="Sacred Flame" type="Archetype Feature" source="Unearthed Arcana: Mages of Strixhaven" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_ELOQUENT_APPRENTICE_WARLOCK_SACRED_FLAME">
+		<supports>UA20210608 Eloquent Apprentice</supports>
+		<requirements>ID_WOTC_UA20210608_ARCHETYPE_WARLOCK_MAGE_OF_SILVERQUILL</requirements>
+		<compendium display="false" />
+		<description>
+			<div class="reference">
+				<div element="ID_PHB_SPELL_SACRED_FLAME" />
+			</div>
+		</description>
+		<sheet display="false" />
+		<spellcasting name="Warlock" extend="true">
+			<extend>ID_PHB_SPELL_SACRED_FLAME</extend>
+		</spellcasting>
+		<rules>
+			<select type="Spell" name="Spell (Inky Shroud)" supports="$(spellcasting:list), 0" default="ID_PHB_SPELL_SACRED_FLAME" spellcasting="Warlock" />
+		</rules>
+	</element>
+
+	<element name="Vicious Mockery" type="Archetype Feature" source="Unearthed Arcana: Mages of Strixhaven" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_ELOQUENT_APPRENTICE_WARLOCK_VICIOUS_MOCKERY">
+		<supports>UA20210608 Eloquent Apprentice</supports>
+		<requirements>ID_WOTC_UA20210608_ARCHETYPE_WARLOCK_MAGE_OF_SILVERQUILL</requirements>
+		<compendium display="false" />
+		<description>
+			<div class="reference">
+				<div element="ID_PHB_SPELL_VICIOUS_MOCKERY" />
+			</div>
+		</description>
+		<sheet display="false" />
+		<spellcasting name="Warlock" extend="true">
+			<extend>ID_PHB_SPELL_VICIOUS_MOCKERY</extend>
+		</spellcasting>
+		<rules>
+			<select type="Spell" name="Spell (Inky Shroud)" supports="$(spellcasting:list), 0" default="ID_PHB_SPELL_VICIOUS_MOCKERY" spellcasting="Warlock" />
+		</rules>
+	</element>
+
+	<element name="Darkness" type="Archetype Feature" source="Unearthed Arcana: Mages of Strixhaven" id="ID_WOTC_UA20210608_ARCHETYPE_FEATURE_MAGE_OF_SILVERQUILL_INKY_SHROUD_WARLOCK">
+		<compendium display="false" />
+		<description>
+			<div class="reference">
+				<div element="ID_PHB_SPELL_DARKNESS" />
+			</div>
+		</description>
+		<sheet display="false" />
+		<spellcasting name="Warlock" extend="true">
+			<extend>ID_PHB_SPELL_DARKNESS</extend>
+		</spellcasting>
+		<rules>
+			<select type="Spell" name="Spell (Inky Shroud)" supports="$(spellcasting:list), $(spellcasting:slots)" default="ID_PHB_SPELL_DARKNESS" spellcasting="Warlock" />
+		</rules>
+	</element>
```

Comment:
These spells should just be granted for the appropriate spellcating ability.  Nothing in Eloquent Apprentice or Inky Shroud implies that they can be swapped later.  Other subclasses like the Divine Soul and the TCoE sorcerers have specific language for swapping the spells.